includes/Loading/DesignSituation.js

```diff
@@ -0,0 +1,461 @@
+/**
+ * Creates design situation object
+ * @param {Number}  no                      Number of design situation, can be undefined
+ * @param {String}  design_situation_type   Design situation type
+ * @param {Object}  params                  Additional parameters, can be undefined
+ * @param {String}  comment                 Comment, can be undefined
+ * @returns     Created design situation object
+ */
+ function DesignSituation (no,
+    design_situation_type,
+    params,
+    comment) {
+    if (!(design_situation_type in design_situation_types)) {
+        console.log("Design type " + design_situation_type + " does not exist");
+        get_design_situation_types();
+    }
+    this.design_situation = createBaseDesignSituation(no, comment, params);
+    this.design_situation.design_situation_type = design_situation_types[design_situation_type];
+}
+
+
+/**
+ * Sets combination wizard
+ * @param {Object}  combination_wizard  Combination wizard
+ */
+DesignSituation.prototype.CombinationWizard = function (combination_wizard) {
+    this.design_situation.combination_wizard = combination_wizard;
+};
+
+
+/**
+ * Sets relationship between load cases
+ * @param {Object} relationship_between_load_cases
+ */
+DesignSituation.prototype.ConsiderInclusiveExclusiveLoadCases = function (relationship_between_load_cases) {
```

Comment:
Better to send just identification of number of relationship_between_load_cases and then check if exist. If yes - use it, if not assert